src/layouts/Extensions.vue

```diff
@@ -1,57 +1,121 @@
 <template>
-	<div v-if="extensions.length > 0" class="extension-list">
-		<p>List of available extensions to use with Tachiyomi, you can download them from here or from the app.</p>
-		<div v-for="extensionGroup in extensions" :key="extensionGroup[0].lang">
-			<h3>
-				{{
-					extensionGroup[0].lang === "en"
-						? simpleLangName(extensionGroup[0].lang)
-						: langName(extensionGroup[0].lang)
-				}}
-				<span class="extensions-total">
-					Total:
-					<span class="extensions-total-sum">
-						{{ extensions.reduce((sum, item) => sum + item.length, 0) }}
-					</span>
-				</span>
-			</h3>
-			<div
-				v-for="extension in extensionGroup"
-				:id="extension.pkg.replace('eu.kanade.tachiyomi.extension.', '')"
-				:key="extension.apk"
-				class="anchor"
-			>
-				<div class="extension">
-					<a
-						:href="`#${extension.pkg.replace('eu.kanade.tachiyomi.extension.', '')}`"
-						@click.stop
-						aria-hidden="true"
-						class="header-anchor"
+	<div class="extension-list">
+		<span class="filters-list">
+			<input
+				type="search"
+				name="extension-search"
+				v-model="filters.search"
+				placeholder="Search extensions by name..."
+				style="text-align: center;"
+			/>
+			<div class="select">
+				<select v-model="filters.lang">
+					<option
+						v-for="[group] in extensions"
+						:key="group.lang"
+						:label="group.lang == 'en' ? simpleLangName(group.lang) : langName(group.lang)"
+						:value="group.lang"
+					></option>
+				</select>
```

Comment:
And here is how you can add a default value as placeholder and it will only work when `filters.lang` is `""`
```suggestion
				<select v-model="filters.lang">
					<option value="" disabled selected>Select lanague</option>
					<option
						v-for="[group] in extensions"
						:key="group.lang"
						:label="group.lang == 'en' ? simpleLangName(group.lang) : langName(group.lang)"
						:value="group.lang"
					></option>
				</select>
```